app/frontend/src/components/forestrysectoropscreening/Landing.vue

```diff
@@ -211,8 +189,8 @@ export default {
   name: 'ForestrySectorLanding',
   data() {
     return {
-      docTitle: 'Protecting Employees, Contractors, and Employers Working in the Silviculture Sector During the COVID-19 Pandemic',
-      docShortTitle: 'COVID-19 Guidelines for Silviculture Work Camps',
+      docTitle: 'Protecting Employees, Contractors, and Employers Working ... During the COVID-19 Pandemic',
```

Comment:
This was Matt's suggestion. I think because the guidelines document still relates specifically to Silviculture guidelines. I'm not sure I understand it either.

Reply:
https://github.com/bcgov/silviculture-ipc/pull/109#pullrequestreview-427587596
i think its thebest we can do until there is  more generic guidelines pdf for all forestry types

Reply:
Sounds good. 👍

Reply:
Yeah that's the intended wording